mathics_scanner/data/named-characters.yml

```diff
@@ -6221,6 +6220,8 @@ RawTilde:
   has-unicode-inverse: false
   is-letter-like: false
   unicode-equivalent: '~'
+  wl-unicode: '~'
```

Comment:
Something for another time. The comments at the top say that if wl-equivalent is the same as ascii, please omit.